bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: made sso configurable via params

## Description
(https://issues.redhat.com/browse/MGDSTRM-8565)

## Verification Steps
make sso/setup and make sso/config should be executed both with and without any parameters to see if it works both ways

## Checklist 
- [x] Relevant tests are passing
